dwave/cloud/computation.py

```diff
@@ -789,8 +792,8 @@ def wait_sampleset(self):
         vartype_from_problem_type = {'ising': 'SPIN', 'qubo': 'BINARY'}
         vartype = vartype_from_problem_type[self.problem_type]
 
-        # include timing and id in info
-        info = dict(timing=self.timing, problem_id=self.id)
+        # include timing and id/label in info
+        info = dict(timing=self.timing, problem_id=self.id, problem_label=self.label)
```

Comment:
maybe better to just not include it if it's `None`?

Reply:
Good catch.

Reply:
FWIW, in current SAPI implementation, `label` is always returned (set to empty string if undefined), so this won't have effect unless/until SAPI behavior is changed.

Reply:
Cannot say I love that. Feels like if it'll always be an empty string on the way out, then it should default to an empty string on the way in, for symmetry is nothing else. But I gather this touches on a deeper discussion with the SAPI team.

Reply:
Yup, after the latest discussion, they'll do:
- `label` can be omitted or set to `null` on input,
- in which case `label` will be set to `null` on output